src/app/tests/suites/certification/Test_TC_DRLK_2_1.yaml

```diff
@@ -321,7 +337,7 @@ tests:
 
           ./chip-tool doorlock read door-open-events 1 1
 
-          Via the TH (chip-tool), verify that the DoorOpenEvents attribute value as 10.
+          Via the TH (chip-tool), verify that the DoorOpenEvents attribute value is 0.
```

Comment:
Can this be updated to be automated? I'm not sure why this is a manual step currently, but the point of the issue was to automate the test fully. The statement in the comment above is incorrect - if the attribute is present, the value is required to be 0 and it looks like someone just made this test pass by basically commenting out this failing step.